### PR TITLE
Pull Request Week 2 for Feb. 2 and 4

### PR DESCRIPTION
**Pull request by mnwilliams2730**
Feb. 2 comfort: 1; Feb. 4 comfort: 3.

### CODE REVIEW

**Comment by mnwilliams2730** [comment]
Looks like my Day 1 homework came with today's too. Don't know how that happened, but at least you have both days now!

File: class03/michelle/disney.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+characters = ["Mickey Mouse", "Aladdin", "Snow White", "Peter Pan", "Simba", "Mulan", 
+	"Goofy", "Elsa", "Tarzan", "Donald Duck", "Shere Khan", "Prince Charming", "Cinderella", "Mufasa", "Belle", "Jiminy Cricket"]
+
+puts characters.sample
+puts characters[7]
```
Comment (mobjohnson) [inline class03/michelle/disney.rb]: I think the intent of this requirement is to use a method to find the element contains ‘Elsa’ as opposed to passing the numeric index. How would you do that?

File: class03/michelle/wines.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#No. 3 -- returns just the white wines
+white_wines = wine_cellar.select do |color|
+  
+end
+
+#No. 4
+
+#No. 5
+
+#No 6
```
Comment (mobjohnson) [inline class03/michelle/wines.rb]: It is always better to submit incomplete homework, than to not submit!!

**Comment by mobjohnson** [comment]
Good Job! I added some comments to your code, but you need not resubmit.

**Pull request merged**